### PR TITLE
Updated to match new namespace.

### PR DESCRIPTION
**Pull request by bhathaway**
I've installed the host package on my machine and successful build using `make example` in the cpp directory.

### CODE REVIEW

File: cpp/README.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ The Unsubscribe functions clean up the socket and queues from a previous Subscri
 
 ```
 ZendarError NextImage(shannon::data::Image& proto, int timeout = -1);
-ZendarError NextTracker(zen::tracker::message::TrackerState& proto,
+ZendarError NextTracker(zpb::tracker::message::TrackerState& proto,
                 Int timeout = -1);
 ZendarError NextTracklog(shannon::data::Position& proto, int = -1);
 ZendarError NextLogMessage(shannon::data::LogRecord& proto, int = -1);
```
Comment (ghsyu) [inline cpp/README.md]: I believe the `shannon::data` namespace is now the `zpb::data` namespace, so this comment should be updated.

**Review by ghsyu — CHANGES_REQUESTED** [review]
Just one comment that got missed because it was already out of date.

File: cpp/README.md (hunk 1 above)
Comment (bhathaway) [reply cpp/README.md]: Cool. Let me take care of that.

**Review by ghsyu — APPROVED** [review]
Ok, looks good to me.

**Pull request merged**